docs/DeveloperGuide.md

```diff
@@ -288,16 +288,48 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `FINDvisor` and the **Actor** is the `user`, unless specified otherwise)
+
+**Use case: Edit a person**
```

Comment:
```suggestion
Use case: Edit a contact
```
Should the subject be contact instead of person? This is to align with the feature specification

Reply:
I think `contact` would be better

Reply:
Reason why I left it as `person` is because the codebase uses `Person` as a class for representing our contact. Not sure if this will cause any implications with respect to documentation. Do leave another reply to confirm whether to change to `contact` or leave it as `person`.

Reply:
I think for use cases using contact sounds more natural, person feels a bit weird. But both would be works actually. Maybe we can add it in the glossary that we use contact and person synonymously.

Reply:
I just realize person is used in the UG too, then I think its better to just standardize and use person everywhere.

Reply:
@KohGuanZeh @Dethada I agree too, let us stick with `person`